File: Unidad 3/Java/Programa6U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa6u3;

import java.util.Scanner;
public class Programa6U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num;
        System.out.println("Programa para saber si un numero es par o impar");
        System.out.println("Introdusca un numero");
        num = sc.nextInt();
        
        if (num%2==0){
            System.out.println("El numero es par");
            
        }else{
        System.out.println("El numero es impar");
        }
        
    }
}

File: Unidad 3/C#/Programa6U3.cs
// See https://aka.ms/new-console-template for more information
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa6U3
    {
        static void Main(string[] args)
        {
            int num;
            Console.WriteLine("Programa para saber si un numero es par o impar");
            Console.WriteLine("Introdusca un numero");
            num = Convert.ToInt32(Console.ReadLine());

            if (num % 2 == 0)
            {
                Console.WriteLine("El numero es par");

            }
            else
            {
                Console.WriteLine("El numero es impar");
            }
        }
    }
}